lib/index.ts

```diff
@@ -36,6 +36,7 @@ export interface PullPushOptions {
 		auth?: string;
 		registrytoken?: string;
 	};
+	abortSignal: AbortSignal;
```

Comment:
I missed this on the review @cywang117. This property should have been optional https://github.com/balena-io-modules/balena-compose/actions/runs/15035039703/job/42255137234